airflow/providers/snowflake/operators/snowflake.py

```diff
@@ -42,7 +42,7 @@ class SnowflakeOperator(SQLExecuteQueryOperator):
         a list of str (sql statements), or a reference to a template file.
         Template references are recognized by str ending in '.sql'
     :param autocommit: if True, each command is automatically committed.
-        (default value: True)
+        (default value: False)
```

Comment:
I suggest to remove it all together (this is what we usually do)
The default value and the description is taken from the parent class

Reply:
Yep.